templates/handlebars/newgroup.handlebars

```diff
@@ -0,0 +1,4 @@
+<div class="views_group_{{i}} col-md-4 col-sm-5 col-lg-4" >
```

Comment:
You need to write col-[size]-[num] only if you change the number on breakpoints, i.e. having both col-lg-4 aand col-md-4 is redundant, as the width doesn't change on md.

Reply:
So which one should be removed?

Reply:
IIRC, `col-lg-4`, make sure by resizing the window

Reply:
Thanks